build/macos-x64/install.dependencies.sh

```diff
@@ -50,16 +50,16 @@ brew uninstall --force little-cms2
 brew uninstall --force libtiff
 brew uninstall --force jasper
 brew uninstall --force libjpeg
-brew uninstall --force jpeg
+brew uninstall --force jpeg-turbo
 brew uninstall --force sox
 brew uninstall --force libpng
 brew uninstall --force curl-openssl
 brew uninstall --force libzip
 brew uninstall --force httpd
-brew uninstall --ignore-dependencies brotli
-brew uninstall --ignore-dependencies libidn2
-brew uninstall --ignore-dependencies xz
-brew uninstall --ignore-dependencies zstd
+brew uninstall --force --ignore-dependencies brotli
```

Comment:
This used to work without the `--force`? Was this added to be consistent with the rest? Do we really need it?

Reply:
Same as the `cmake` change here - on my new VM that I set up these 4 packages don't exist after a fresh install of homebrew and so the `brew uninstall` command fails. `--force` ensures that the command runs even if the dependency isn't installed

Reply:
I would prefer to keep this as as. Since we are using the hosted GitHub actions builder.